### PR TITLE
add frozen version of requirements.txt that will be more stable

### PR DESCRIPTION
**Pull request by nsinkov**
## Changes proposed in this PR

- add frozen version of requirements.txt

## Why are we making these changes?

frozen version of requirements.txt that will be more stable when building. A change to a package in pypi can cause an install to fail

### CODE REVIEW

File: integration/requirements.txt [hunk 1]
```diff
@@ -1,15 +1,24 @@
+# NOTE: make sure to update the requirements-frozen.txt file when making a change here
+# Sort this file lexicographically
+# Generate the requirements-frozen.txt file as follows:
+#  pip install -r requirements.txt
+#  pip freeze > requirements-frozen.txt
```
Comment (dposada) [inline integration/requirements.txt:5]: Does this need to be in a "clean" environment, like a fresh virtualenv where I haven't already pip installed other stuff not needed for the integration tests?

File: integration/requirements-frozen.txt [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+ufw==0.36
+urllib3==1.22
+virtualenv==20.0.25
+wcwidth==0.2.5
+webencodings==0.5.1
+widgetsnbextension==3.5.1
+zipp==3.1.0
+zope.interface==4.3.2
+file:../cli#egg=cook_client
+pip==9.0.1; python_version >= '3.6'
```
Comment (dposada) [inline integration/requirements-frozen.txt:110]: Should we use this file in places where we currently use `requirements.txt`? e.g. the Vagrantfile, Dockerfiles, etc.?

File: integration/requirements.txt (hunk 1 above)
Comment (nsinkov) [reply integration/requirements.txt:5]: it doesn't really matter. if you get a green run it's good.

Comment (nsinkov) [reply integration/requirements.txt:5]: The key thing is that every dependency is frozen

File: integration/requirements-frozen.txt (hunk 2 above)
Comment (nsinkov) [reply integration/requirements-frozen.txt:110]: updated

File: integration/requirements.txt (hunk 1 above)
Comment (dposada) [reply integration/requirements.txt:5]: If I have pip installed a bunch of extra garbage that the tests don't need, they will get included in this file, right? That seems wrong.

Comment (nsinkov) [reply integration/requirements.txt:5]: I don't think it matters much if the tests run, but I'll say to do it from Vagrant

Comment (nsinkov) [reply integration/requirements.txt:7]: updated the instructions. you were right - I was able to pull things in that at least Travis couldn't install. that's a problem

**Pull request closed without merging**